include/openmc/physics.h

```diff
@@ -101,6 +105,26 @@ void sample_secondary_photons(Particle& p, int i_nuclide);
 //! \param[in] p, particle to be split or rouletted with the weight window.
 void split_particle(Particle& p);
 
+#ifdef NCRYSTAL
+//==============================================================================
+// NCrystal wrapper class for the OpenMC random number generator
+//==============================================================================
+
+class NCrystalRNGWrapper : public NCrystal::RNGStream {
+  uint64_t* openmc_seed_;
```

Comment:
This should probably be in the `private:` section?

Reply:
Well, it technically is since the default access level of classes is private :-)

But we can certainly make an explicit `private:` section if you prefer.

Reply:
Thanks! Yes, we generally follow the C++ core guidelines when possible:
[NL.16: Use a conventional class member declaration order](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-order)

Reply:
So this was done now.